.github/workflows/state-compatibility-check.yml

```diff
@@ -110,15 +110,15 @@ jobs:
           SNAPSHOT_ID=$(echo $SNAPSHOT_INFO | dasel --plain -r json  '(file=osmosis-1-pre-epoch).filename' | cut -f 1 -d '.')
 
           # Download snapshot if not already present
-          mkdir -p $HOME/snapshots/$REPO_MAJOR_VERSION/
-          if [ ! -d "$HOME/snapshots/$REPO_MAJOR_VERSION/$SNAPSHOT_ID" ]; then
-              rm -rf $HOME/snapshots/$REPO_MAJOR_VERSION/*
-              mkdir $HOME/snapshots/$REPO_MAJOR_VERSION/$SNAPSHOT_ID
-              wget -q -O - $SNAPSHOT_URL | lz4 -d | tar -C $HOME/snapshots/$REPO_MAJOR_VERSION/$SNAPSHOT_ID -xvf -
+          mkdir -p /mnt/data/snapshots/$REPO_MAJOR_VERSION/
+          if [ ! -d "/mnt/data/snapshots/$REPO_MAJOR_VERSION/$SNAPSHOT_ID" ]; then
+              rm -rf /mnt/data/snapshots/$REPO_MAJOR_VERSION/*
+              mkdir /mnt/data/snapshots/$REPO_MAJOR_VERSION/$SNAPSHOT_ID
+              wget -q -O - $SNAPSHOT_URL | lz4 -d | tar -C /mnt/data/snapshots/$REPO_MAJOR_VERSION/$SNAPSHOT_ID -xvf -
           fi
 
           # Copy snapshot in Data folder
-          cp -R $HOME/snapshots/$REPO_MAJOR_VERSION/$SNAPSHOT_ID/data $HOME/.osmosisd/
+          cp -R /mnt/data/snapshots/$REPO_MAJOR_VERSION/$SNAPSHOT_ID/* $HOME/.osmosisd/
```

Comment:
This line fixes a bug that was happening on `v11` where only the data folder and not the `wasm` folder would be copied into the osmosis `data` folder